discord/permissions.py

```diff
@@ -396,9 +396,9 @@ def send_tts_messages(self) -> int:
     def manage_messages(self) -> int:
         """:class:`bool`: Returns ``True`` if a user can delete or pin messages in a text channel.
```

Comment:
Preemptively, since we'll probably forget by the time this rolls out fully 
```suggestion
         """:class:`bool`: Returns ``True`` if a user can delete messages in a text channel.
```